alembic/versions/2018022007_add_columns_cpuset_and_gpuset__19b1e7f3608b.py

```diff
@@ -0,0 +1,24 @@
+"""Add columns cpuset and gpuset to worker_run table
```

Comment:
Why do we have both this migration that adds cpuset/gpuset and the one that replaces cpus/gpus?

Reply:
This one keeps track of cpu/gpusets occupied per bundle ; the other one is for storing cpu/gpusets offered by the worker.